bioconductor-scater-scripts-post-install-tests.sh

```diff
@@ -81,6 +82,10 @@ export return_log='TRUE'
 export log_exprs_offset='NULL'
 export centre_size_factors='TRUE'
 export return_norm_as_exprs='TRUE'
+export feature_controls='NULL'
+export cell_controls='NULL'
+export nmads='5'
```

Comment:
Don't need to quote these. R will receive them as strings anyway, but opt_parse will make them numbers again.